game2.html

```diff
@@ -20,54 +21,26 @@
       <li>If your guess is too high the game will inform you "You haven't lived that long yet."</li>
       <li>If you guess more than 20 times you lose!</li>
       <li>Once you enter your age correctly in hours you've won!</li>
+      <li> <a href="#two">Click here for examples</a></li>
     </ol>
-      <p><a href="#bottom">Click here for examples</a></p>
-      <figure>
+    </p>
+    <figure>
+      <p id="results"><strong>Your results will be displayed here.</strong></p>
```

Comment:
FYI, since you are inserting text (w/ .textContent) into the p tag, it will overwrite the strong element within. If you want to want to retain the strong element, you could move the id='results' into the strong tag.